internal/jobservice/repository/job_table.go

```diff
@@ -0,0 +1,19 @@
+package repository
```

Comment:
I would suggest putting only the interface in the repository package, and locate the implementations outside.

Reply:
See comment below.  I tried to do this but I think that the exporting gets tricky and I'm unsure if we really need a repostiryo package for different implementations.